### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by lgrahl**

### CODE REVIEW

File: android/app/src/main/java/org/saltyrtc/demo/app/MainActivity.java [hunk 1]
```diff
@@ -404,7 +404,7 @@ public void sendDc(View view) {
 		Log.d(LOG_TAG, "Sending message...");
 		final String text = this.textInput.getText().toString();
 		final ByteBuffer bytes = StandardCharsets.UTF_8.encode(text);
-		this.sdc.send(new DataChannel.Buffer(bytes, true));
+		this.sdc.send(new DataChannel.Buffer(bytes, false));
```
Comment (dbrgn) [inline android/app/src/main/java/org/saltyrtc/demo/app/MainActivity.java]: Why not binary mode?

**Review by dbrgn — COMMENTED** [review]
IIRC the reason why the public key is in there is that it also acts as documentation for the public key that needs to be used in the webapp. But if you think it's not needed, feel free to merge!

File: android/app/src/main/java/org/saltyrtc/demo/app/MainActivity.java (hunk 1 above)
Comment (lgrahl) [reply android/app/src/main/java/org/saltyrtc/demo/app/MainActivity.java]: Oops. That was just for testing.

**Comment by lgrahl** [comment]
I'll add the derived public key to *Key Info*. That should be sufficient for documentation. :)

**Comment by lgrahl** [comment]
Depends on https://github.com/saltyrtc/saltyrtc-client-java/pull/71

**Review by dbrgn — APPROVED** [review]
Whether the key is derived or not doesn't matter for me.

**Pull request merged**